cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -429,7 +429,7 @@ void ThinClientPoolDM::cleanStaleConnections(std::atomic<bool>& isRunning) {
       }
     }
 
-    size_t replaceCount =
+    int replaceCount =
```

Comment:
As long as you're changing the declaration of replaceCount, please make it 'auto' rather than an explicit type.

Reply:
Sure